e2e/nx-run/src/affected-graph.test.ts

```diff
@@ -309,8 +309,8 @@ describe('Nx Affected and Graph Tests', () => {
       runCLI(`generate @nx/web:app ${myapp}`);
       runCLI(`generate @nx/web:app ${myapp2}`);
       runCLI(`generate @nx/web:app ${myapp3}`);
-      runCLI(`generate @nx/js:lib ${mylib}`);
-      runCLI(`generate @nx/js:lib ${mylib2}`);
+      runCLI(`generate @nx/js:lib ${mylib} --no-interactive`);
```

Comment:
What causes various x-prompt's to not show up? We should listen to the same thing in the new prompt (or better yet set NX_INTERACTIVE to false when it would be triggered) to avoid having to update all of these.

Reply:
That's a good point. The helper doesn't prompt if `NX_INTERACTIVE=false`, so something else is disabling the `x-prompt` from the schema. I'll look into that.

I initially thought about setting `NX_INTERACTIVE=false` for all commands. As I see it, we always need to run all e2e tests in non-interactive mode. So, we might also consider setting that in the `runCli` and `runCLiAsync` helpers.